css/organization-page.styl

```diff
@@ -54,6 +58,9 @@
   min-height: 400px
   position: relative
 
+  @media (max-width: 900px)
+    min-height: 500px
```

Comment:
not thrilled with this solution, fits all of Snapshot Safari's hero elements, but might not fit all of a different org's. unsure of alternate solution.